app.js

```diff
@@ -68,4 +68,13 @@ let sum1 = 0
 for (let i = 11; i <= 30; i += 2) {
 	sum1 += i;
 }
-console.log(sum1);
\ No newline at end of file
+console.log(sum1);
+
+console.log("==============Challenge 8=================");
+// Coding challenge #8: Create a function that will convert from Celsius to Fahrenheit
+
+let celcius = 0
+
+function celciusToFahrenheit(celcius) {
```

Comment:
you should add `= 0`  as a default parameter

Reply:
Roger.